ops/ccs-ops-misc/load_test/README.md

```diff
@@ -98,6 +98,8 @@ Essentially, all the items you would set up in the config file are set in a sing
 
 **--workerThreads** : (Optional) : Controls running in distributed mode; If this is set to >1, will spawn that many worker threads to run the tests across multiple cores. If not provided, defaults to 1. See section on running in distributed mode for more info.
 
+**--resetStats** : (Optional) : If this flag is included, the test statistics will reset to zero after clients have finished spawning. Note that this option does *not* reset the test run time, so you should extend the run time by however long it will take to spawn all clients. (Example: the default 100 clients at 5 clients per second will take 20 seconds to spawn, so you should adjust the test run time from 1m to 1m20s). **Note:** There are many reasons why we might want to capture statistics while new load is being added. There might be performance problems accepting the connection or new connections might affect users already connected to the system.
```

Comment:
It's a bit onerous to ask the user to extend the run time when using this; since the ramp up time is calculable, can we just automatically add that runtime when --resetStats is used? (And adjust the documentation to note that the ramp up time will be added to the total runtime, but excluded from the stats)

Reply:
On second thought, maybe just always increase the test time by the calculated ramp up time, so that the total test is run for the expected input time; even if we want to include the ramp up time in the stats, we still want the test to run for the specified time at full capacity.

Reply:
Yeah, that's very do-able, and it's one less thing for the user to try to keep track of. I'll take care of that, and I'll make sure the extended test time is documented as well.